nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/ConsumeKafka_2_6.java

```diff
@@ -484,8 +484,10 @@ public void onTrigger(ProcessContext context, ProcessSession session) throws Pro
                     + "Will roll back session and discard any partially received data.", lease);
             } catch (final KafkaException kex) {
                 getLogger().error("Exception while interacting with Kafka so will close the lease {} due to {}", lease, kex, kex);
+                context.yield();
```

Comment:
Processor should give a little time to service UI requests on connection failures.